.travis.yml

```diff
@@ -28,6 +28,11 @@ matrix:
       script:
         - src/build/ci-test.sh
       jdk: oraclejdk9
+    - env:
+        - TESTS=Y
```

Comment:
Couldn't these environment variables be moved into the respective `ci-*.sh` scripts?

Reply:
I was just copy and pasting what was already here ;)